opts/envfile_test.go

```diff
@@ -29,7 +29,12 @@ func TestParseEnvFileGoodFile(t *testing.T) {
 
 _foobar=foobaz
 `
-
+	// Adding a newline + a line with pure whitespace.
+	// This is being done like this instead of the block above
+	// because it's common for editors to trim trailing whitespace
+	// from lines, which becomes annoying since that's the
+	// exact thing we need to test.
+	content += "\n    "
```

Comment:
Ah, I see the misunderstanding, but it says "trim the line first" NOT "trim the first line". ;)

tl;dr, if there's a line with _only_ whitespace, docker would error before.

> Do we need a similar test, using a tab as whitespace?

I could just add a `\t` into this line as well to cover both.